releasenotes/notes/42580.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: promotion
+area: istioctl
+releaseNotes:
+  - |
+    **Fixed** Use Context instead of noctx.
```

Comment:
Not sure a release note is indeed necessary as it's internal but maybe mention the replacement of a deprecated `DialTLS` with `DialTLSContext`?

Reply:
@ymesika It doesn't seem necessary. This is a test file.

Reply:
You can remove this file now with the `release-notes-none` label.